atlp-rwanda/elites-bn-fe
# #181339591-likes and dislike accommodations

# What does this PR do?
Users should be able to like/unlike an accommodation facility

# Description of Task to be completed?

- [x] : Create Like and dislike button
- [x] : display number of likes and dislike for a certain page (accommodation).
- [x] : Done with Tests 

# How should this be manually tested?
Start login through the link below : https://elites-barefoot-fe-git-dev-elites-team.vercel.app/
1. Like and dislike, you have to select an accommodation and then check on view an accommodation
2. After Check on the thumb up icons  for like or thumb down icon for dislike.
3. After you will be able to see number of likes or dislikes incremented or decremented.

## How Has This Been Tested?
https://elites-barefoot-fe-git-dev-elites-team.vercel.app/

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration


# Any background context you want to provide?

# What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/181339591 

# Screenshots (if appropriate)

# Questions:
N/A
